commands/auth.md

```diff
@@ -1,21 +1,20 @@
 The AUTH command authenticates the current connection in two cases:
 
-1. If the Redis server is password protected via the `requirepass` option.
-2. A Redis 6.0 instance, or greater, is using the [Redis ACL system](/topics/acl).
+1. If the Valkey server is password protected via the `requirepass` option.
+2. If the Valkey server is using the [Valkey ACL system](/topics/acl).
 
-Redis versions prior of Redis 6 were only able to understand the one argument
-version of the command:
+Valkey understands the one argument version of the command:
```

Comment:
From a doc perspective, I would explain `AUTH <username> <password>` first, and then say `AUTH <password>` is just a shorthand for `AUTH default <password>`. Without the versioning information, this seems weird.\

Reply:
Ah, sorry, it's about the sentence affected by this change. (Reading too fast.)